AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Request/GetURL/AWSStorageGetURLOptions.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Provides options spectific to the AWS S3 plugin implementation of the Storage category.
+///
+/// Usage example:
+///
+/// ```
+/// // Because `validateObjectExistence` is set to true, the
+/// // underlying plugin will throw and error if no S3 object
+/// // can be resolved with the given key.
+/// let url = try await Amplify.Storage.getURL(
+///     key: "ExampleKey",
+///     options: .init(
+///         accessLevel: .private,
+///         pluginOptions: S3GetUrlPluginOptions(
+///             validateObjectExistence: true
+///         )
+///     )
+/// )
+/// ```
+///
+/// See: [StorageListRequestOptions.pluginOptions](x-source-tag://StorageListRequestOptions.pluginOptions)
+///
+/// - Tag: AWSStorageGetURLOptions
+public struct AWSStorageGetURLOptions {
```

Comment:
This name follows the `AWS<Category Operation>Options` pattern (note that the AWS service name is not in it) precedent set by: https://github.com/aws-amplify/amplify-swift/tree/main/AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Models/Options